src/main/java/net/atos/zac/app/planitems/converter/RESTPlanItemConverter.java

```diff
@@ -46,6 +47,8 @@ public RESTPlanItem convertPlanItem(final PlanItemInstance planItem, final UUID
         restPlanItem.naam = planItem.getName();
         restPlanItem.zaakUuid = zaakUuid;
         restPlanItem.type = convertDefinitionType(planItem.getPlanItemDefinitionType());
+        restPlanItem.formulierDefinitie =
```

Comment:
De assignmengt van restPlanItem.formulierDefinitie is alleen van toepassing op een Human task. Deze assignemnt moet daarom naar de convertHumanTask methode waar het eerst ook stond

Reply:
aangepast